manifest-rhel-9.2.yaml

```diff
@@ -108,6 +108,11 @@ postprocess:
      ---
      EOF
 
+     # Tweak dependencies of systemd-user-sessions.service to not rely on network.
+     # Copy file to /etc because /usr is a read-only filesystem.
```

Comment:
No.  In this case we're doing this at build time to *generate* the content in `/usr`.

Reply:
(This is a fundamental difference between doing a PR for this repository versus e.g. https://github.com/openshift/machine-config-operator/ - the latter case is "configuration" that is lifecycled outside of the OS and lives in `/etc` or `/var` mostly; the content here goes in `/usr`)

In a [layering](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/coreos-layering.html#coreos-layering) world, this is unified because it's equally easy to drop higher level content in `/usr` too.

Reply:
Oh this is great, makes it even easier for me then. Just fixed and now I modify directly in `/usr`